### ISSUE TITLE
QUESTION: It seems that the SmallMap has a lack of basic traits.

### ISSUE THREAD

**Issue by jjeffcaii** [ihciah/small-map#2]
For example, `Debug`, `Clone`, `Eq`... I mean, when the key/values implements the traits above.

**Comment by ihciah** [comment]
Yes it is. This project is still in very eraly stage. There are many methods not implemented. Would you like to implement some?
